triggers/pixelbinEvents.js

```diff
@@ -0,0 +1,383 @@
+var hookID = "";
+const eventIds = [];
+
+const subscribeHook = async (z, bundle) => {
```

Comment:
rename this to PixelbinAllEvents.js, add comment for the same, with reason on why it is like this

Reply:
name: `(${bundle.meta.zap.id})-${uuidv4()}`,